### PR TITLE
Talos IPv4 feed

### PR DESCRIPTION
**Pull request by DctrG**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Talos IPv4 blocklist feed

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Please edit the integration according to my comments and then we'll schedule a demo.

Let me know if you have any questions.

File: Packs/FeedTalos/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "FeedTalos",
+    "description": "Talos Threat Intelligence IPv4 Feed",
+    "support": "xsoar",
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/pack_metadata.json:4]: This is to be supported by Content, I did the same as in my previous contributions, or did something in the procedure change?

File: Packs/FeedTalos/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/pack_metadata.json:6]: Same as above

File: Packs/FeedTalos/TestPlaybooks/FeedTalos_Test.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
+id: FeedTalosTestPlaybook.yml
+name: FeedTalosTestPlaybook.yml
+version: -1
+fromversion: 4.5.0
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/TestPlaybooks/FeedTalos_Test.yml]: Changed.

File: Packs/FeedTalos/TestPlaybooks/FeedTalos_Test.yml [hunk 4]
```diff
@@ -0,0 +1,95 @@
+id: FeedTalosTestPlaybook.yml
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/TestPlaybooks/FeedTalos_Test.yml]: Done

File: Packs/FeedTalos/TestPlaybooks/FeedTalos_Test.yml [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+name: FeedTalosTestPlaybook.yml
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/TestPlaybooks/FeedTalos_Test.yml]: Done

File: Tests/conf.json [hunk 6]
```diff
@@ -2970,6 +2970,11 @@
             "playbookID": "FailedInstances - Test",
             "integrations": "Whois",
             "fromversion": "4.5.0"
+        },
+        {
+            "integrations": "Talos Feed",
+            "playbookID": "TalosFeed_Test",
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Tests/conf.json]: Done

File: Packs/FeedTalos/Integrations/FeedTalos/FeedTalos.yml [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/Integrations/FeedTalos/FeedTalos.yml]: Interesting, no idea how did happened, could it be something with the sdk? I have removed them manually.

File: Packs/FeedTalos/pack_metadata.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (barchen1) [reply Packs/FeedTalos/pack_metadata.json:4]: yes, it has been changed.
If you want this pack to be supported by xsoar you need to open an issue about it

[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/pack_metadata.json]: Understood. I've made the changed and pushed the code

File: Packs/FeedTalos/Integrations/FeedTalos/FeedTalos.py [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+    Returns:
+        Indicators.
+    """
+    iterator = client.build_iterator()
+    indicators = []
+    if limit > 0:
+        iterator = iterator[:limit]
+    for item in iterator:
+        value = item.get('value')
+        type_ = item.get('type', FeedIndicatorType.Domain)
```
Comment (barchen1) [inline Packs/FeedTalos/Integrations/FeedTalos/FeedTalos.py:91]: Please change as we talked in the demo

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks for the demo!
Here are the things we talked about:
1. Run demisto-sdk secrets and add the secrets to secrets_ignore file.
2. Need a better error handling for invalid url parameter.
3. Change the url parameter location to be above the Indicator Reputation parameter.

Please let me know if you have any questions.

File: Packs/FeedTalos/Integrations/FeedTalos/FeedTalos.yml [hunk 9]
```diff
@@ -82,6 +82,16 @@ configuration:
...
```
Comment (barchen1) [inline Packs/FeedTalos/Integrations/FeedTalos/FeedTalos.yml:94]: Looks like it happened again, please fix it like you did last time.
Screen Shot 2020-09-16 at 10 24 14

**Review by barchen1 — CHANGES_REQUESTED** [review]

File: Packs/FeedTalos/Integrations/FeedTalos/FeedTalos_description.md [hunk 10]
```diff
@@ -1,3 +1,7 @@
+### Community Contributed Integration
+#### Integration Author: Guillermo Serrano
+Support and maintenance for this integration are provided by the author. Please use the following contact details:
+***
 ## Talos IP Threat Feed
 
 Collects indicators from Talos Threat Intelligence IPv4 feed:
```
[the remark this answers is not in the document]
Comment (DctrG) [reply Packs/FeedTalos/Integrations/FeedTalos/FeedTalos_description.md]: The whole file empty? By the way, the first 4 lines must have been added by the SDK...

**Review by barchen1 — APPROVED** [review]

**Pull request merged**